### DISCUSSION TITLE
[PSD1] Exercise 1.12

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#117] [Esercizi (A-L)]
**Exercise 1.12**

What is the smallest (most negative) 32-bit binary number that can be represented with: 
	(a) unsigned numbers?
	(b) two’s complement numbers?
	(c) sign/magnitude numbers?

**Comment by Elia-Belli** [comment]
Il più piccolo numero binario a 32bit rappresentabile con le seguenti codifiche è:

- Unsigned: $0$
- Complemento a 2:  $-2^{31}$
- Sign/magnitude: $-2^{31}+1$
